### PR TITLE
npsim.py: PFRICH -> RICHEndcapN, remove MRICH

### PR DESCRIPTION
**Pull request by veprbl**
### Briefly, what does this PR introduce?

Detector was renamed in https://github.com/eic/epic/pull/664

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [ ] New feature (issue #__)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
Yes

### Does this PR change default behavior?
No

### CODE REVIEW

File: src/dd4pod/python/npsim.py [hunk 1]
```diff
@@ -49,14 +49,12 @@ def setupCerenkov(kernel):
   )
   # This could probably be a substring
   SIM.filter.mapDetFilter['DRICH'] = 'opticalphotons'
-  SIM.filter.mapDetFilter['MRICH'] = 'opticalphotons'
-  SIM.filter.mapDetFilter['PFRICH'] = 'opticalphotons'
+  SIM.filter.mapDetFilter['RICHEndcapN'] = 'opticalphotons'
```
Comment (wdconinc) [inline src/dd4pod/python/npsim.py:52]: I think this does a substring match, so just a single line would work too (needs verification):
```
  SIM.filter.mapDetFilter['RICH'] = 'opticalphotons'
```

Comment (veprbl) [reply src/dd4pod/python/npsim.py:52]: And let's not rely on that

Comment (wdconinc) [reply src/dd4pod/python/npsim.py:52]: Even better this wouldn't be in npsim at all, of course, but I haven't figured out a scalable way of doing that.

**Review by wdconinc — APPROVED** [review]

**Comment by veprbl** [comment]
Should we tag 1.4.1 with this?

**Comment by wdconinc** [comment]
> Should we tag 1.4.1 with this?

No, no one looks at this anyway, or wont this month. It won't be a problem if it takes a month before this is active.

**Comment by veprbl** [comment]
Yes, nightly please :)

**Pull request merged**